.github/workflows/release.yaml

```diff
@@ -0,0 +1,75 @@
+name: Weekly Release
+
+on:
+  #  Uncoment these two lines to test the pipeline in a PR, but NEVER merge in main branch:
+  #  pull_request:
+  #    branches: [ "master" ]
+  schedule:
+    - cron: '0 11 * * 1' # Run every Monday at 7am NYC time
+      branches:
+        - master
+  workflow_dispatch: # allows manual releasing
+      branches:
+        - master
+jobs:
+  build_and_release:
+    runs-on: ubuntu-latest
+    # Uncomment for testing in PRs, but NEVER merge in main branch:
+    #if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Bazel
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '21'
+
+      - name: Build with Bazel
+        run: bazel build java/com/google/copybara/copybara_deploy.jar
```

Comment:
I think this needs `--stamp` or `--embed_label` here, since today's new release prints
```
Feb 07, 2025 3:52:34 PM com.google.copybara.Main configureLog
INFO: Setting up LogManager
Copybara source mover (Version: Unknown version)
```

Reply:
I have to look at this, our current oss implementation does this:
```
  protected String getVersion() {
    return "Unknown version";
  }
```
I have to match to what we do internally.

Reply:
Just merged https://github.com/google/copybara/commit/eee4901b216793e392c1d38de416abcc4969e8d5 for this